open-api/rest-catalog-open-api.yaml

```diff
@@ -549,28 +549,57 @@ paths:
     post:
       tags:
         - Catalog API
-      summary: Returns either a list of `PlanTask`, a list of `FileScanTask`. If planning is not complete returns a `plan-id`.
+      summary: Submit a scan for planning
       description:
-        Prepares either a list of `PlanTask` that can be used to distribute table scan planning, or a list of `FileScanTask`, based on a set of table scan criteria
-        such as selected columns, filters, snapshot range, case sensitivity, etc.
-        In the event that the plan tasks or file scan tasks are not ready to be served, the service will return a `plan-id`,
-        which can be used as input in the `GetTasksStatus`.
-        
-        Requires that the client specifies only a `snapshot-id` for a snapshot scan, or for performing incremental scans only provide 
-        a `start-snapshot-id` and an `end-snapshot-id`. 
-      
-        Each `PlanTask`, can be used as input in the `RetrieveTasks` API
-        to request a subset of file scan tasks in a table scan.
-        This mechanism allows clients to distribute and parallelize the entire table scan planning process.
-      operationId: planTable
+        Submits a scan for server-side planning.
+
+
```

Comment:
I've verified that the spacing in these descriptions renders correctly in the Swagger Editor.

Reply:
[nit] I think we need only one new line between the two paragraphs